web/src/engine/websites/EighteenComic.ts

```diff
@@ -0,0 +1,81 @@
+import { Tags } from '../Tags';
+import icon from './EighteenComic.webp';
+import { Chapter, DecoratableMangaScraper, Page, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchHTML, FetchWindowScript } from '../platform/FetchProvider';
+import type { Priority } from '../taskpool/DeferredTask';
+import DeScramble from '../transformers/ImageDescrambler';
+function MangaInfoExtractor(element: HTMLDivElement) {
+    return {
+        id: element.querySelector<HTMLAnchorElement>('a').pathname,
+        title: element.querySelector<HTMLImageElement>('img').alt.trim()
+    };
+}
+
+const pageScript = `
+    new Promise ( resolve => {
+        const imagenodes = [...document.querySelectorAll('div.row.thumb-overlay-albums img')];
+        const scrambled = aid > scramble_id;
+        resolve ( imagenodes.map(image => {
+            const imageNumber = image.parentNode.id.split('.')[0];
+            const seed = scrambled ? get_num(window.btoa(aid), window.btoa(imageNumber)) : -1;
```

Comment:
May return `0` instead of `-1` to avoid confusion that `0` could be a valid seed